storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3071,10 +3071,9 @@ static int rocksdb_init_func(void *const p) {
                    MY_MUTEX_INIT_FAST);
 
 #if defined(HAVE_PSI_INTERFACE)
-  rdb_collation_exceptions =
-      new Regex_list_handler(key_rwlock_collation_exception_list);
+  rdb_collation_exceptions = new Regex(key_rwlock_collation_exception_list);
```

Comment:
By now I am thoroughly confused whether MyRocks code is using MySQL code style or not, IIRC I have seen upstream pull requests moving both directions